pkgs/top-level/aliases.nix

```diff
@@ -1522,6 +1522,7 @@ mapAliases {
   xonsh-unwrapped = python3Packages.xonsh; # Added 2024-06-18
   xplayer = throw "xplayer has been removed as the upstream project was archived"; # Added 2024-12-27
   xprite-editor = throw "'xprite-editor' has been removed due to lack of maintenance upstream. Consider using 'pablodraw' or 'aseprite' instead"; # Added 2024-09-14
+  xsv = throw "'xsv' has been removed due to lack of upstream maintenance";
```

Comment:
Could we perhaps alias to xan or nudge the user to use it? Functionality-wise, xan should provide everything xsv used to.

Reply:
IMHO it makes more sense to get this PR in first and change the alias in your xan PR. We haven't packaged xan yet. :)